index.html

```diff
@@ -257,10 +257,10 @@ <h3>Skills</h3>
       </section>
 
       <section class="form-contact">
-        <h3 class="text-contact">
+        <p class="text-contact">
           I'm always interested in hearing about new projects, so if you'd like
           to chat please get in touch.
-        </h3>
+        </p>
         <form
           id="template-form"
           action="https://formspree.io/f/mbjwkvzr"
```

Comment:
Please ensure that every form control has a label associated with it using 'label', 'for', and 'id'.